molgenis/molgenis-emx2
# feat: can create a form with only subset of columns. Use case in catalogue3

When editing a form, this feature enables to set that only a subset of the columns are editable.

To test this:
* download 'catalogueontologies' and 'vac4eu' from https://vac4eu.molgeniscloud.org/
* create a new 'catalogue' schema called 'vac4eu' 
* upload the downloaded files

Then you can open vac4eu/catalogue3 -> navigate to 'DatasetMappings' link at bottom of screen -> choose a datasetMapping to test with

E.g. on my system
http://localhost:8080/VAC4EU/catalogue/#/dataset-mappings/SDO/SDO/ConcePTION_CDM/VISIT_OCCURRENCE
when you click the 'pencil' you will notice that you can only edit subset of the columns

N.b. there is interaction with another bugfix, showing the lookup labels wrongly in the first select, see #2172
